data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2757,6 +2772,25 @@
         [/message]
     [/event]
 
+    # After attacking Malifor 3 times, Father Morvin realizes he might be able to destroy Malifor
+    [event]
+        name=attack end
+        first_time_only=no
+        [if]
+            [variable]
+                name=malifor_attacked
+                equals=3
+            [/variable]
+            [then]
+                [message]
+                    speaker=Father Morvin
+                    message= _ "Let me and Thera attack him. We might be the only ones capable of destroying him!"
+                [/message]
+            [/then]
+        [/if]
+    [/event]
```

Comment:
Not sure what the CI failure is complaining about here. One too many blank lines between the `[event]` blocks?